README.md

```diff
@@ -10,6 +10,22 @@ The sources of the news articles
 ## Dataset
 Dataset used to analyze clickbait articles and train the classifier
 
+
+Article JSON format:
+```json
+{
+	"id" : "<id>",
+	"name": "<headline>",
+	"url" : "<url>",
+	"timestamp" : "<date time of publication>",
```

Comment:
For when you start making data in this format, you may want to look up how JS treats timestamps - https://stackoverflow.com/questions/10286204/the-right-json-date-format#15952652

JSON seems to not really care, so it will be a string for JSON, but putting in this format will make the FSW people happier.